src/stylesheets/components/_accordions.scss

```diff
@@ -90,17 +86,14 @@ $v-padding: $spacing-md-small;
   margin: 0;
 
   &:not(:first-child) {
-    margin-top: $spacing-x-small;
+    margin-top: units(0.5);
```

Comment:
See comment on line 70

Reply:
The Sketch file is using a gap of 10px for this, do we want to bump this up to 8px/1 unit?

Reply:
Sounds good